packages/vx-demo/src/next.config.js

```diff
@@ -1,43 +1,38 @@
 /* eslint-disable no-undef */
-const withCss = require('@zeit/next-css');
```

Comment:
👍 I think initially I needed this because the `sandbox`s need css to work on codesandbox, but I think I moved these to the `index` (which does `ReactDOM` stuff) so they aren't referenced in `vx-demo` anymore